code/game/atoms.dm

```diff
@@ -486,6 +486,10 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 	SendSignal(COMSIG_ATOM_EMAG_ACT)
 	return
 
+/atom/proc/rad_act(strength)

```

Comment:
this is retarded, just call SendSignal on the atom

Reply:
This proc already exists and is used down the inheritance chain

Reply:
since you're rewriting radiation you can get rid of the proc

Reply:
Alright, time for a bunch of new components then

Reply:
HA RADtarded i just got that